src/main/java/org/ga4gh/starterkit/wes/utils/runmanager/language/NextflowLanguageHandler.java

```diff
@@ -76,7 +76,9 @@ public NextflowLanguageConfig getLanguageConfig() {
 
     public String[] constructWorkflowRunCommand() throws Exception {
         if (!validWorkflowFound()) {
-            throw new Exception("A valid workflow could not be determined from the workflow URL");
+            Exception e = new Exception("A valid workflow could not be determined from the workflow URL");
```

Comment:
I would revert this as it's basically the same logic as before. If we need to add logs, then maybe we can add the, just before throw new Exception line.
Example:
```
loggingUtil.error("Exception occurred. Message: A valid workflow could not be determined from the workflow URL");
throw new Exception("A valid workflow could not be determined from the workflow URL");
```

you could also add more details to the exception, if you have access them.

Reply:
It is helpful to add logs where we catch and then throw the exception with a generic message. For example,
line 95 in [this file](https://github.com/ga4gh/ga4gh-starter-kit-wes/blob/logging/src/main/java/org/ga4gh/starterkit/wes/utils/requesthandler/GetRunLogRequestHandler.java)

I would do add a log that captures the original exception message.
```
loggingUtil.error("Exception occurred: Could not load WES run log. Original Exception message: " + ex.getMessage());
throw new BadRequestException("Could not load WES run log");
```